airbyte-integrations/connectors/destination-snowflake/src/test-integration/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestinationTest.java

```diff
@@ -1,29 +0,0 @@
-/*
- * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
- */
-
-package io.airbyte.integrations.destination.snowflake;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-import com.fasterxml.jackson.databind.JsonNode;
-import io.airbyte.commons.json.Jsons;
-import io.airbyte.protocol.models.AirbyteConnectionStatus;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import org.junit.jupiter.api.Test;
-
-class SnowflakeDestinationTest {
```

Comment:
I'm guessing this file shouldn't be deleted?

Reply:
actually SnowflakeDestinationTest class was not deleted but renamed to SnowflakeDestinationIntegrationTest because SnowflakeDestinationTest have been already present in **destination-snowflake** module(test) 
![snow01](https://user-images.githubusercontent.com/39538064/147232249-6b2fc99b-c2c3-49d2-aa0b-c4cfb36fa53c.png)